vars/dockerBinaryBuild.groovy

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/groovy
 
-def call(String name, String version, String dockerHubOrg, String dockerHubRepo = name, String credentialId = "dockerhubjenkins", String fromDir = './docker', Closure body = {}) {
+def call(String name, String version, String dockerHubOrg, String dockerHubRepo = name, String credentialId = "dockerhubjenkins", String fromDir = './docker', String dockerfilePath = '.', Closure body = {}) {
```

Comment:
Wouldn't this mean, that everywhere we set the body closure, we would need to set the dockerfilePath as well? But that might currently only be millicore.

Reply:
If you specify a closure, you would need to specify everything. That said we don't use the closure anywhere anymore since https://github.com/feedhenry/fh-pipeline-library/pull/49 and i was tempted to remove it as an option at that time as i see no reason you would ever need to use it. Millicore no longer needs it https://github.com/fheng/millicore/blob/master/Jenkinsfile#L103

Reply:
In that case, lets remove the closure.